lhotse/augmentation/__init__.py

```diff
@@ -0,0 +1,7 @@
+import torchaudio as _torchaudio
+
+from .common import AugmentFn
+from .wavaugment import *
+
+if str(_torchaudio.__version__) >= '0.7.0':
```

Comment:
FYI: If `torchaudio` hits `0.10.0` release in the future (we do not know if we will move to the major 1.0 release or not), this could produce a wrong result.

```
$ python
>>> '0.7.0' > '0.10.0'
True
```

A future-proof way would be to split the version string and compare major version and minor version as number.

Reply:
Well spotted!

Reply:
`packaging.version` may be helpful in this case.
See https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python